understanding/20/non-text-content.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Non-text Content</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Non-text information is available to more people</dd>
-		<dt>Author task</dt><dd>Create a text alternative for visual and auditory content</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Non-text information is available to more people.</dd>
+         <dt>What to do</dt><dd>Create a text alternative for visual and auditory content.</dd>
+         <dt>Why it's important</dt><dd>People who can’t fully see or hear content can understand it.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>So assistive technologies (which can only understand text) can present the information in speech, braille, simplified language or other form that some users need.</dd> 
```

Reply:
@GreggVan also suggested
```suggestion
         <dt>Why it's important</dt><dd>So non-text content is available in text where it can be converted and presented to users in a form they can understand (e.g. braille, speech, morse code, simpler language)</dd> 
```

Reply:
In general, the 'why' sections focus on the users. The humans build the motivation for authors to take action.
I suggest to keep tat focus consistently.
The technical details are good to mention in the long understanding part.